src/MessageBox/__stories__/MessageBox.stories.js

```diff
@@ -0,0 +1,163 @@
+        <div className='fddocs-container'>
+            <Button onClick={() => setShowConfirmationMessageBox(true)}>Confirmation</Button>
+            <Button onClick={() => setShowErrorMessageBox(true)} type='negative'>Error</Button>
+            <Button onClick={() => setShowSuccessMessageBox(true)} type='positive'>Success</Button>
+            <Button onClick={() => setShowWarningMessageBox(true)} type='attention'>Warning</Button>
+            <Button onClick={() => setShowInformationMessageBox(true)}>Information</Button>
+
+            <MessageBox
+                actions={[
+                    (<Button compact option='emphasized'>Yes</Button>),
+                    (<Button compact>No</Button>)
+                ]}
+                onClose={() => setShowConfirmationMessageBox(false)}
+                show={showConfirmationMessageBox}
+                title='Confirmation'
+                type='confirmation'>
+                <p>Are you sure?</p>
+            </MessageBox>
+
+            <MessageBox
+                actions={[
+                    (<Button compact>OK</Button>)
+                ]}
+                onClose={() => setShowErrorMessageBox(false)}
+                show={showErrorMessageBox}
+                title='Error'
+                type='error'>
+                <p>This is an error message!</p>
+            </MessageBox>
+
+            <MessageBox
+                actions={[
+                    (<Button compact>OK</Button>)
+                ]}
+                onClose={() => setShowSuccessMessageBox(false)}
+                show={showSuccessMessageBox}
+                title='Success'
+                type='success'>
+                <p>This is a success message.</p>
+            </MessageBox>
+
+            <MessageBox
+                actions={[
+                    (<Button compact>OK</Button>)
+                ]}
+                onClose={() => setShowWarningMessageBox(false)}
+                show={showWarningMessageBox}
+                title='Warning'
+                type='warning'>
+                <p>This is a warning!</p>
+            </MessageBox>
+
+            <MessageBox
+                actions={[
+                    (<Button compact>OK</Button>)
+                ]}
+                onClose={() => setShowInformationMessageBox(false)}
+                show={showInformationMessageBox}
+                title='Information'
+                type='information'>
+                <p>This is some information.</p>
+            </MessageBox>
+        </div>
+    );
+};
+
+export const sizes = () => {
+    const [showMessageBoxSm, setShowMessageBoxSm] = useState(false);
+    const [showMessageBoxMd, setShowMessageBoxMd] = useState(false);
+    const [showMessageBoxLg, setShowMessageBoxLg] = useState(false);
+    const [showMessageBoxXl, setShowMessageBoxXl] = useState(false);
+
+
+    return (
+        <div className='fddocs-container'>
+            <Button onClick={() => setShowMessageBoxSm(true)}>Small</Button>
```

Comment:
I cannot really see the differences between those sizes

Reply:
They effectively only change the left/right margin size. I do myself feel they're not very useful, but fundamental-styles provides them, so...

Reply:
That's true... It's not what I would expect 😕